gatling-core/src/main/resources/gatling-defaults.conf

```diff
@@ -77,6 +77,7 @@ gatling {
     noReports = false       # When set to true, don't generate HTML reports
     maxPlotPerSeries = 1000 # Number of points per graph in Gatling reports
     useGroupDurationMetric = false  # Switch group timings from cumulated response time to group duration.
+    printActualValue = false   # When set to true, will print actual value of the assertion even if the assertion passes.
```

Comment:
No need for this option, let's print the actual value anyway

Reply:
sounds good. Updated!